### ISSUE TITLE
Preferences treeview cannot be navigated with NVDA

### ISSUE THREAD

**Issue by sukiletxe** [microsoft/vscode#80633]
Code Version: 1.38.0
NVDA version: 2019.2
Windows version: Windows 7 SP 1
Steps:
Open preferences and switch to browse mode (ctrl+ space). Notice there is a tree view at the bottom. Try switching to browse mode (ctrl+space or enter) when you have moved to it with the arrow keys. Enter doesn't work, and when pressing ctrl+ space the arrow keys don't work either. Object Navigation does not help either because it only shows some of the items. Tabbing is possible, but with the man y settings there are it is somewhat impractical, though filtering helps a bit. Also, when tabbing, sometimes NVDA goes silent, and when changing combo box values the arrows don't echo selected choices. The only way to use combo boxes is to use NVDA + Numpad Enter to open the combo box.

**Comment by sukiletxe** [comment]
Not CTRL+Space but NVDA+Space, sorry.

**Comment by isidorn** [comment]
Also feels like one of the duplicates @roblourens

**Comment by roblourens** [comment]
I think this feedback is covered in the other settings editor accessibility issues. Thanks for the feedback!
